### DISCUSSION TITLE
提示有特洛伊木马……

### DISCUSSION THREAD

**Discussion by debbieqiu** [mikumifa/biliTickerBuy#161] [提问专区]
![619e5a9c8eb6055dcd76cc6630475299](https://github.com/mikumifa/biliTickerBuy/assets/25548837/bd3b7df0-0823-4195-bce5-70ba183b33ec)

**Comment by mikumifa** [comment]
没看懂

**Comment by ZianTT** [comment]
有病毒，你要不别用了吧🤣
